### PR TITLE
Bug/id 169/time to live randomly of incidents

### LINKED ISSUE #169: Time to live of incident generated are not being created randomly
**Describe the bug**
The lifetime of the generated incidents is the same, therefore, it is not being generated randomly

**To Reproduce**
1. Open Hermes App
2. Click on the button to generate incidents
3. Set the range and number of incidents
4. In the incident handler click on "display"
5. See the time to live of different incidents

**Expected behavior**
The lifetime of each incident should be generated randomly.

**Screenshots**
Time to live set
Time to live set 2

**Smartphone:**
 - Device: Emulator Pixel 6 API 33
 - OS: Android

### PR DESCRIPTION
**Pull request by JorgeHB69**
**What?**
The incidents generated randomly now have a different time to live.
Also, the algorithm was improved to don't generate the same incident at the same place.

**How?**
Changed a parameter to be initialized inside the method.

**Why?**
Because if the method is initialized outside the method, the previous values maintain on the next iteration.

### CODE REVIEW

**Comment by JorgeHB69** [comment]
https://github.com/CampusDelSaber/hermes/issues/169

**Review by SamuelEscalera — APPROVED** [review]
Everything is okey

**Review by Elvis-Castro-H — APPROVED** [review]
Approved

File: app/src/test/java/com/isc/hermes/generators/IncidentGeneratorTest.java [hunk 1]
```diff
@@ -23,8 +33,23 @@ public void initSetup() {
...
+        incidents.forEach(incident -> {
+            if (incident.getGeometry().getType().equals("Polygon")) {
+                System.out.println(incident.getType());
+                System.out.println(incident.getReason());
+                printPolygon("polygon", incident.getGeometry().getCoordinates());
+            }
+        });
+    }
+
+    private void printPolygon(String message, List<Double[]> coordinates) {
```
Comment (luiggy-mamani-jalafund) [inline app/src/test/java/com/isc/hermes/generators/IncidentGeneratorTest.java]: remove all the logs for the test

File: app/src/test/java/com/isc/hermes/generators/IncidentGeneratorTest.java [hunk 2]
```diff
@@ -23,8 +33,23 @@ public void initSetup() {
 
     @Test
     public void incidentGeneratedTest() {
-        assertNotNull(incidentGenerator
-                .getIncidentsRandomly(coordinateReference, Radium.FIFTY_METERS, 2));
+        List<Incident> incidents = incidentGenerator
+                .getIncidentsRandomly(coordinateReference, Radium.FIFTY_METERS, 5);
+        incidents.forEach(incident -> {
+            if (incident.getGeometry().getType().equals("Polygon")) {
+                System.out.println(incident.getType());
```
Comment (luiggymamani) [inline app/src/test/java/com/isc/hermes/generators/IncidentGeneratorTest.java:40]: remove logs

File: app/src/test/java/com/isc/hermes/generators/IncidentGeneratorTest.java [hunk 3]
```diff
@@ -40,4 +65,23 @@ public void timeToWaitTest() {
         assertTrue(incidentGenerator.getMinutesBetweenTimeout() > 0);
         assertTrue(incidentGenerator.getSecondsBetweenTimeout() > 0);
     }
+
+    @Test
+    public void multiPointTest() {
+        PointGenerator pointGenerator = new PointGenerator();
+        pointGenerator.getMultiPoint(coordinateReference, Radium.ONE_KILOMETER, 3).forEach(incident ->
+                System.out.println(Arrays.toString(incident)));
```
Comment (luiggymamani) [inline app/src/test/java/com/isc/hermes/generators/IncidentGeneratorTest.java:73]: remove logs

**Review by luiggy-mamani-jalafund — COMMENTED** [review]
Bug fixed successfully
Screenshot_20230703-090200_Hermes

**Review by mmoralesv — APPROVED** [review]

**Review by luiggy-mamani-jalafund — APPROVED** [review]

**Pull request merged**